Serenity updates - October 2020 - Momo's news (#786)
akeneo/pim-helpcenter: content/updates/2020-10/8-New-filters-for-rules-conditions.md

---
pim_announcement_img: ./img/new-filters-for-rules-conditions.png
pim_announcement_alt_img: "The new filters in action"
pim_announcement_audience:
- EE
---

# New filters for rules conditions
::: meta-data type="New" features="Automation, Productivity" available="early October" link-to-doc="../articles/manage-your-rules.html#available-conditions"
:::

We added new filters in the rules engine to help you better define your conditions. You can now filter on:

> - the entity type: to target product models or products and product variants,

Reviewer: ```suggestion
- the entity type: to target product models or products and variant products,
```

::: meta-data type="New" features="Automation, Productivity" available="early October" link-to-doc="../articles/.html"
:::

In the Rules engine, create more efficient rules conditions thanks to these 4 new filters:
- On the entity type, i.e., to choose whether to target product models or products,
- On family variants,

> - On product or product model parents,

Reviewer 2: ```suggestion
- parents: to target specific product parents or products without parents
```
I'd like to explain the use case for each filter.. but it's not that easy...

Author: Yes, that's what I try to do at first, it was tricky, so I stopped.

> - On identifiers.

Reviewer 2: ```suggestion
- identifiers: to target specific products based on their product identifiers.
```

![The new filters in action](../img/TODO.png)

> Remember that the more precise your rule condition is, the more performant your rule will be. So don't hesitate to overuse those new filters. 😉

Reviewer 2: ```suggestion
Condition filters enable you to narrow the product selection to apply the rules on, so the more precise your rule conditions are, the faster your rule will be run. So don't hesitate to use those new filters.
```
